packages/eas-cli/src/utils/progress.ts

```diff
@@ -29,7 +30,7 @@ export function createProgressTracker({
   let transferredSoFar = 0;
   let current = 0;
 
-  const timerLabel = String(Date.now());
+  const timerLabel = String(randomUUID());
```

Comment:
I'm just curious: why is it needed?

Reply:
Sure! I was getting errors that "a timer is being ended which hasn't been started" or something like that. This fixed it. I think the two promises (for uploading eas.json and project archive) were starting and finishing too quickly and, thus, having same `Date.now()`, conflicting with each other?